interpreter/interpreter_test.go

```diff
@@ -791,23 +766,57 @@ var (
 			},
 		},
 		{
-			name: `ExprBench/false_1st`,
+			name: `false_1st`,
 			E:    `false && true`,
-			I:    map[string]interface{}{},
+			I:    EmptyActivation(),
 		},
 		{
-			name: `ExprBench/false_2nd`,
+			name: `false_2nd`,
 			E:    `true && false`,
-			I:    map[string]interface{}{},
+			I:    EmptyActivation(),
+		},
+		{
+			name: "conditional",
+			E:    `a ? b < 1.2 : c == ['hello']`,
+			Env: []*exprpb.Decl{
+				decls.NewIdent("a", decls.Bool, nil),
+				decls.NewIdent("b", decls.Double, nil),
+				decls.NewIdent("c", decls.NewListType(decls.String), nil),
+			},
+			I: map[string]interface{}{
+				"a": false,
+				"b": 2.0,
+				"c": []string{"hello"},
+			},
+		},
+		{
+			name: "exists_literal",
+			E:    `[1, 2, 3, 4, 5u, 1.0].exists(e, type(e) == uint)`,
+			I:    EmptyActivation(),
 		},
 		{
-			name: `ExprBench/complex`,
+			name: "exists_variable",
+			E:    `elems.exists(e, type(e) == uint)`,
+			Env: []*exprpb.Decl{
+				decls.NewIdent("elems", decls.NewListType(decls.Dyn), nil),
+			},
+			I: map[string]interface{}{
+				"elems": []interface{}{0, 1, 2, 3, 4, uint(5), 6},
+			},
+		},
+		{
+			name: `complex`,
```

Comment:
Super nitpick, but can you change the backticks to double quotes (or the other way around)?

Reply:
Sure